server/api/links.js

```diff
@@ -0,0 +1,44 @@
+const router = require('express').Router();
+const Err = require('../db').model('err');
+const Link = require('../db').model('link');
+const err_link = require('../db').model('err_link');
+const Promise = require('bluebird');
+
+router
+
+.get('/', (req, res, next) => {
+  Link.findAll()
+  .then(links => res.send(links))
+  .catch(next);
+})
+
+// get all links associated with a particular error
+.get('/:errId', (req, res, next) => {
```

Comment:
Otherwise, the request looks like `GET /api/links/872465` -> "get this specific link"